src/bsp/platform/stm32f4/xi_bsp_rng_stm32f4.c

```diff
@@ -6,21 +6,64 @@
 
 #include <xi_bsp_rng.h>
 
+#define BSP_RNG_DEBUG_LOG 1
```

Comment:
This is good! Do you think we could make it more generic ? I'm thinking about putting this "common bsp logging" code into some *.h file and start using it across the bsp's. What do you guys think ?